src/checks/spatial/buddy_check.rs

```diff
@@ -161,7 +191,20 @@ mod tests {
     fn test_buddy_check() {
         assert_eq!(
             buddy_check(
-                &SpatialCache::new(
+                &[
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.),
+                    Some(0.1),
+                    Some(1.)
+                ]
+                .to_vec(),
```

Comment:
Clippy suggests using `as_ref()` instead of `to_vec()`

Reply:
fixed